api/database/factories/RevisionMocks.php

```diff
@@ -34,7 +34,7 @@
         $tags = factory(App\Models\Tag::class, $faker->numberBetween(2, 4))->make();
         $article['tags'] = $tags->lists('tag')->toArray();
 
-        $meta = $factory->raw(App\Models\ArticleMeta::class);
+        $meta = $factory->raw(App\Models\Meta::class);
```

Comment:
As meta isn't being tracked now, we probably shouldn't have this here in the mock